leavestracker/apps/leaves/serializers.py

```diff
@@ -0,0 +1,16 @@
+from leavestracker.apps.leaves.models import Leaves
+from rest_framework import serializers
+
+class LeavesSerializer(serializers.ModelSerializer):
+    first_name = serializers.ReadOnlyField(source='user.first_name')
+    last_name = serializers.ReadOnlyField(source='user.last_name')
+    email = serializers.ReadOnlyField(source='user.email')
+
+    class Meta:
+        model = Leaves
+        fields = ( 'start_date','end_date','reason', 'first_name', 'last_name', 'email')

```

Comment:
Do we not need to pass leave id ?

Reply:
we don't use at FE right now so I didn't pass it.